packages/flutter/lib/src/animation/animation_controller.dart

```diff
@@ -348,7 +348,17 @@ class AnimationController extends Animation<double>
   /// after this method is called.
   @override
   void dispose() {
+    assert(() {
+      if (_ticker == null) {
+        throw new FlutterError(
+          'AnimationController.dispose() called more than once.\n'
+          'A given AnimationController for $debugLabel cannot be disposed more than once.'
```

Comment:
It'd still be useful to mention the this toString (which includes debugLabel) in the message (also in the other one in this patch, for Route or Navigator or whatever it is).